examples/incremental_learning/helmet-detection-modelbox/incremental_learning-modelbox/thirdparty/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+#
```

Comment:
It is better not to directly copy thirdparty source file here, please try to make these as dependency.
You can consider to move them to Dockerfile.